src/geoops/combine.jl

```diff
@@ -45,10 +45,9 @@ macro combine(object::Symbol, exprs...)
 end
 
 function _combine(data::D, names, columns) where {D<:Data}
-  dom   = domain(data)
   table = values(data)
 
-  newdom = Collection([centroid(dom)])
+  newdom = Collection([Multi(collect(domain(data)))])
```

Comment:
I will add constructors for Multi with iterators so that we can remove the collect here.

Reply:
Ok, it is now available in Meshes.jl master